ihmc-avatar-interfaces/build.gradle.kts

```
@@ -27,6 +27,9 @@ mainDependencies {
    api("us.ihmc:ihmc-simulation-toolkit:source")
    api("us.ihmc:ihmc-robot-data-visualizer:source")
    api("us.ihmc:robot-environment-awareness-visualizers:source")
+
+   api("com.badlogicgames.gdx-controllers:gdx-controllers-core:2.2.3")
+   api("com.badlogicgames.gdx-controllers:gdx-controllers-desktop:2.2.3")
```

I would add a comment and you don't need to declare core.
```suggestion
   api("com.badlogicgames.gdx-controllers:gdx-controllers-desktop:2.2.3") // for CSG plugin
```